frontend/src/lib/components/TaxonomicFilter/InfiniteSelectResults.tsx

```diff
@@ -66,6 +70,79 @@ function CategoryPill({
     )
 }
 
+function TaxonomicGroupTitle({ openTab }: { openTab: TaxonomicFilterGroupType }): JSX.Element {
+    const { taxonomicGroups } = useValues(taxonomicFilterLogic)
+
+    const { eventOrdering } = useValues(taxonomicFilterPreferencesLogic)
```

Comment:
Would this remember the setting across browser sessions, like LocalStorage?  We have a couple hundred events, so would be nice if, using the same browser, I didn't have to change it to alphabetical each time.

Reply:
It does - it doesn't follow you between devices, it just stores the setting in local storage

this line https://github.com/PostHog/posthog/pull/33184/files#diff-fe3f27d699f80789264fbb9e75e12232133c33175aa13d59c34739f9e54dcf7fR15 stores the setting in local storage whenever it changes

Reply:
Awesome!